source/common/network/cidr_range.h

```diff
@@ -135,6 +135,7 @@ class IpList {
   IpList() = default;
 
   bool contains(const Instance& address) const;
+  int getIpListSize() const { return ip_list_.size(); };
```

Comment:
I don't think this is needed; revert

Reply:
probuf config has already the function to get the repeated size.  Here it is a common requirement to get the size of IpList.  And also in the function tlsKeyLogMatch(), it will use the size to check whether the local/remote is configured or not.